### ISSUE TITLE
Better separate each contribution in final result

### ISSUE THREAD

**Issue by Cwpute** [JamesOsborn-SE/eat-poop-you-cat-android#23]
We played our game without constraints and gave the phone to whomever was available and wanted to play. Which ended up in a bunch of contributions, and that was realy fun !

The game results could have been better presented though, as it seemed that images were much more interesting than descsiptions.

There should be a way to better separate each contribution. Maybe make each of them appear as its own "speach bubble" ? in its own little pannel ?
To better show the progression of the game, maybe insert an arrow that goes from the previous to the next contribution. Time of day at which they contributed could be fun too for longer games.

At least have all the contributions numbered (during the game AND in the final result view) so that we can call each other out when reviewing contributions "hey who contributed as number 8 ?"

**Comment by JamesOsborn-SE** [comment]
I don't think I added the issue yet but I was given the suggestion to allow users put in their names. 

"To better show the progression of the game, maybe insert an arrow that goes from the previous to the next contribution." Do you mean have it look like a flow chart?

**Comment by Cwpute** [comment]
Not quite like a flowchart, but you get the idea yes :)

Le 4 janvier 2024 06:00:12 GMT+01:00, James Osborn ***@***.***> a écrit :

**Comment by JamesOsborn-SE** [comment]
I'm still struggling to understand what you mean. Can you make a drawing showing what you mean?

**Comment by Cwpute** [comment]
![explanation drawing +hour](https://github.com/JamesOsborn-SE/eat-poop-you-cat-android/assets/80604512/48720ad1-5f22-48de-b01a-be50ed542e1a)

**Comment by JamesOsborn-SE** [comment]
This looks more like a text messaging app but just for the sentence part. The tricky part for me is that it looks like the sentence is associated more with the drawing above, where in reality it's connected to the drawings before and after.

**Comment by JamesOsborn-SE** [comment]
I'll leave this here for reference. 
https://stackoverflow.com/questions/7974847/android-drawable-speech-bubble

**Comment by Cwpute** [comment]
It's connected with both actualy.
Yes it does look a bit like a messaging app, but at its core EPYC is a communication game isn't ? ;) i don't think it's out of place.
I will try to draw a new mock-up soon.

Le 2 avril 2024 18:18:42 GMT+02:00, James Osborn ***@***.***> a écrit :

**Comment by Cwpute** [comment]
![explanation drawing test2](https://github.com/JamesOsborn-SE/eat-poop-you-cat-android/assets/80604512/8e533ddc-88a0-4e4f-8078-4f96edb0e2f0)
i'm not sure of the result, but the idea with this one was having speech bubbles coming up from the left, and drawings coming up from the right (the little things are tupposed to be pens).
You're right that the colors may imply each text only corresponds to a single drawing, especially with the colors i chose. I don't really know how to do it otherwise, to let people visually assume it's also linked to the following drawing, and i'm not sure it's too much of an issue.

**Comment by Cwpute** [comment]
Yeah, after some consideration, i think i prefer my first mockup best: it's efficient and simple enough.
I think whether people will perceive the text as refering to both the previous and next drawing or not will com down to how much blank space we leave between each contribution.
